### ISSUE TITLE
Outline feature doesn't pick up on functions that are parts of tables. (Lua)

### ISSUE THREAD

**Issue by buddyberg** [nesbox/TIC-80#2026]
If you create a table and then create functions that are a part of that table, the outline will not realize that they are there. For example:

```lua
local draws={}
function draws.start()
 -- does not show in outline
end
function drawsstart()
 -- does show in outline
end
```

**Comment by notbored** [comment]
Yeah the code editor only shows top-level functions in the outline, but I agree it would be nice to be able to see nested functions while writing code using the TIC-80's editor.

If you really want to have that now, I suggest using an external editor/IDE for the time being, most have that functionality built into them.

**Comment by TimotheeGreg** [comment]
That's related to #1474
